code/reports/ForumReport.php

```diff
@@ -3,46 +3,49 @@
  * Forum Reports.
  * These are some basic reporting tools which sit in the CMS for the user to view.
  * No fancy graphing tools or anything just some simple querys and numbers
- * 
+ *
  * @package forum
  */
 
 /**
  * Member Signups Report.
- * Lists the Number of people who have signed up in the past months categorized 
+ * Lists the Number of people who have signed up in the past months categorized
  * by month.
  */
 class ForumReport_MemberSignups extends SS_Report {
-	
+
 	public function title() {
 		return _t('Forum.FORUMSIGNUPS', 'Forum Signups by Month');
 	}
-	
+
 	public function sourceRecords($params = array()) {
 		$membersQuery = new SQLQuery();
-		$membersQuery->setFrom('Member');
-		$membersQuery->selectField('DATE_FORMAT("Created", \'%Y %M\')', 'Month');
-		$membersQuery->selectField('COUNT("Created")', 'Signups');
-		$membersQuery->setGroupBy('Month');
-		$membersQuery->setOrderBy('Created', 'DESC');
+		$membersQuery->setFrom('"Member"');
+		$membersQuery->setSelect(array(
+			'Month' => DB::getConn()->formattedDatetimeClause('"Created"', '%Y-%m'),
```

Comment:
One day we relly should create a datetime specific SearchFilter.
```php
array(
	`Created:Year` => $year,
	`Created:Month:LessThan` => $month
)
```

Or something. :)